vault/mount.go

```diff
@@ -1267,6 +1267,9 @@ func (c *Core) requiredMountTable() *MountTable {
 		UUID:             sysUUID,
 		Accessor:         sysAccessor,
 		BackendAwareUUID: sysBackendUUID,
+		Config: MountConfig{
```

Comment:
This will only take effect for new Vault installs.

Reply:
Added 				coreMount.Config = requiredMount.Config
In the case when https://github.com/hashicorp/vault/blob/27cf6a6ae19d18e8ea0b5cdc2c5259e5fcd1d7ba/vault/mount.go#L844 is true
This will override current MountOptions for required backends to the value provided in requiredMountTable()

As theses backend are not directly tunable by the users, it makes sense to me, to always enforce it

WDYT @jefferai ?

Reply:
Seems fine.